test/Rentals.spec.ts

```diff
@@ -3908,4 +3908,70 @@ describe('Rentals', () => {
       await expect(rentals.connect(owner).unpause()).to.be.revertedWith('Pausable: not paused')
     })
   })
+
+  describe('returnToLessor :: ExtendedRentals', () => {
```

Comment:
IMO  It is not a good practise to test mock contracts that are not a copy of what we will have in production

Reply:
Do you suggest the removal of the test or moving it elsewhere?

Reply:
I'll leave the test here. 
We could later discuss a way in which we could standardize/handle cases like this were testnet might have additional "helper" functions

Reply:
Ok, then we should follow the good practice and remove the tests. Here is not the place to keep test mocked smart contracts

Reply:
Sure